code/workspaces/infrastructure-api/src/kubernetes/deploymentGenerator.js

```diff
@@ -5,9 +5,13 @@ import config from '../config/config';
 
 const containerInfo = imageConfig();
 
+function returnImage(type, version) {
```

Comment:
I'd probably expect this to be called `getImage`, but that might just be personal preference!